dags/inference/jetstream_inference_e2e.py

```diff
@@ -94,7 +94,12 @@
           model_configs["scan_layers"] = sweep_model_configs["scan_layers"]
           model_configs["dataset"] = sweep_model_configs["dataset"]
           model_configs["weight_dtype"] = sweep_model_configs["weight_dtype"]
-          model_configs["tokenizer"] = sweep_model_configs["tokenizer"]
+          tokenizer = sweep_model_configs["tokenizer"]
+          # Let gcs path be directly used, else use maxtext/assets dir
+          if not tokenizer.startswith("gs://"):
+            model_configs["tokenizer"] = f"assets/{tokenizer}"
```

Comment:
```suggestion
            model_configs["tokenizer"] = f"maxtext/assets/{tokenizer}"
```

Reply:
This would break the maxengine_server.py command, which uses tokeniser path without 'maxtext' in it. I updated the config to create separate paths for both commands.